hack/local-slack-bot.sh

```diff
@@ -44,8 +44,13 @@ done
 
 os::log::info "Installing the proxy's certificate..."
 http_proxy=localhost:7777 https_proxy=localhost:7777 wget --output-document "${data}/mitm.pem" mitm.it/cert/pem
-sudo cp "${data}/mitm.pem" /etc/pki/ca-trust/source/anchors/
-sudo update-ca-trust
+
+if [ "$(uname -s)" != "Darwin" ]; then
+  sudo cp "${data}/mitm.pem" /etc/pki/ca-trust/source/anchors/
+  sudo update-ca-trust
+else # special logic for MacOS
+  sudo security add-trusted-cert -d -r trustRoot -k /Library/Keychains/System.keychain "${data}/mitm.pem"
```

Comment:
nit: `sudo` looks demanding (and I do not see `remove-trusted-cert` in tear down) but it was there before the PR.
Local dev. env. is hard. I know.